### PR TITLE
move a few resources around, all img src's use the image libary now

### PR DESCRIPTION
**Pull request by devongillis**
most changes are just shuffling image resources around, other files had one or two lines changed to now use the image library

### CODE REVIEW

File: poker-pals/src/images/ImageFiles.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+module.exports =  {
+
+    getImage(name){
+        let image = this.imageFiles.find(image => image.name === name);
+        if(image === undefined){
+            return this.imageFiles.find(image => image.name === 'not_found');
```
Comment (chankk) [inline poker-pals/src/images/ImageFiles.js]: Can I ask what the purpose of this is?

Comment (devongillis) [reply poker-pals/src/images/ImageFiles.js]: which part? the function?

Comment (devongillis) [reply poker-pals/src/images/ImageFiles.js]: the function provides an easier way for  to access resources

Comment (chankk) [reply poker-pals/src/images/ImageFiles.js]: The purpose of an image that displays when you can't find the image you want.

Comment (devongillis) [reply poker-pals/src/images/ImageFiles.js]: oh, i was trying to figure out a weird behaviour, the app was showing that an image was found but then it rendered as if the image was not there (i figured it out), I'll remove it now

**Review by chankk — APPROVED** [review]

**Review by rzayevan — APPROVED** [review]

**Pull request merged**